### PR TITLE
Update gemspec version

### PR DESCRIPTION
**Pull request by lmd59**
Pull requests into cqm-parsers require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

**Review by dczulada — APPROVED** [review]

File: cqm-parsers.gemspec [hunk 1]
```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |s|
   s.authors = ["The MITRE Corporation"]
   s.license = 'Apache-2.0'
 
-  s.version = '3.1.0.0'
+  s.version = '3.1.1.0'
 
   s.add_dependency 'cqm-models', '~> 3.0.0'
```
Comment (adongare) [inline cqm-parsers.gemspec:14]: Can we get latest models in as well or would that be more work & testing? I had pulled it in bonnie_version but not in master.

**Review by adongare — APPROVED** [review]
